src/pages/DebuggerContent.tsx

```diff
@@ -80,12 +81,34 @@ const DebuggerContent = () => {
                         dispatch(setIsProgramInvalid(true));
                       }
 
+                      function tryAsSpi(program: number[]) {
+                        try {
+                          const { code, memory, registers } = Program.fromSpi(
+                            new Uint8Array(program),
+                            new Uint8Array(),
+                          );
+                          const regs = Array.from(registers.getAllU64()) as RegistersArray;
+                          return {
+                            program: Array.from(code) || [],
+                            initial: { ...initialState, regs, mem: memory },
+                            name: "custom",
+                          };
+                        } catch {
```

Comment:
Can we make `try` a bit more minimal? I guess the only thing that is expected to throw is `Program.fromSpi`, isn't it?